lucene/facet/src/java/org/apache/lucene/facet/Facets.java

```diff
@@ -29,6 +29,12 @@ public abstract class Facets {
   /** Default constructor. */
   public Facets() {}
 
+  /**
+   * Returns all the children labels with non-zero counts under the specified path in the unsorted
+   * order. Returns null if the specified path doesn't exist or if this dimension was never seen.
+   */
```

Comment:
Sorry to be picky about this, but since it's part of the externally facing API/javadoc, I think it's important to scrutinize a little bit. What do you think of:

```suggestion
  /**
   * Returns all child labels with non-zero counts under the specified path. Users should make no 
   * assumptions about ordering of the children. Returns null if the specified path doesn't exist or 
   * if this dimension was never seen.
   */
```